emails/tests/models_tests.py

```diff
@@ -237,6 +238,23 @@ def test_valid_address_dupe_of_deleted_invalid(self):
             relay_address.address, relay_address.domain_value
         )
 
+    def test_free_user_cant_set_block_list_emails(self):
+        relay_address = RelayAddress.objects.create(user=self.user)
+        try:
+            relay_address.block_list_emails = True
+            relay_address.save()
+        except ValidationError:
+            relay_address.refresh_from_db()
+            assert relay_address.block_list_emails == False
```

Comment:
This test will pass if the exception is not raised. Please use `assertRaises` to ensure `ValiationError` is raised:

```suggestion
        relay_address.block_list_emails = True
        with self.assertRaises(ValidationError):
            relay_address.save()
        relay_address.refresh_from_db()
        assert relay_address.block_list_emails == False
```